packages/transformers/babel/src/BabelTransformer.js

```diff
@@ -1,8 +1,11 @@
 // @flow strict-local
 
-import {generate, babelErrorEnhancer} from '@parcel/babel-ast-utils';
+import {babelErrorEnhancer} from '@parcel/babel-ast-utils';
 import {Transformer} from '@parcel/plugin';
+import {relativeUrl} from '@parcel/utils';
+import SourceMap from '@parcel/source-map';
 import semver from 'semver';
+import generate from '@babel/generator';
```

Comment:
> I guess this isn't using astring anymore because that doesn't support JSX?

Yes. And TypeScript. And other pre-standard JS features.

> Should it still try to use astring when possible?

How would we know this? I guess we'd have to traverse the AST to find out?